joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/adminfaces/AdminfacesProperties.java

```diff
@@ -69,6 +69,11 @@
 	 */
 	private Boolean renderMessages;
 
+	/**
+	 * When true, p:message will not show details if equals to summary.
+	 */
+	private Boolean skipDetailIfEqualsSummary;
```

Comment:
Could you change adminFaces attribute from **skipDetailIfEqualsSummary** to **skipMessageDetailIfEqualsSummary** in order to use the same AdminFaces naming convention?